docs/sql-error-conditions.md

```diff
@@ -1299,6 +1299,12 @@ Cannot call function `<functionName>` because named argument references are not
 
 It is not allowed to use an aggregate function in the argument of another aggregate function. Please use the inner aggregate function in a sub-query.
 
+### NON_FOLDABLE_ARGUMENT
+
+[SQLSTATE: 22024](sql-error-conditions-sqlstates.html#class-22-data-exception)
```

Comment:
Data exception category is also reasonable to me if there is no other proper one.